ocrd_tesserocr/recognize.py

```diff
@@ -141,7 +163,9 @@ def process(self):
                 #tessapi.SetImage(page_image)
                 
                 LOG.info("Processing page '%s'", page_id)
-                regions = page.get_TextRegion()
+                regions = itertools.chain.from_iterable(
```

Comment:
How is this different from just `[page.get_TextRegion()] + [subregion.get_TextRegion() for  subregion in page.get_TableRegion()]`?

Reply:
Ah, right, it zips the array, so it's `[region1, [subregions of region 1 if any], region2, [subregion of region2 if any]...]`, right?

Reply:
No, the opposite, it's meant to flatten the lists.